CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Django Field Audit change log
+
+## v1.0.0 - 2022-05-31
```

Comment:
Is this level of detail necessary considering this code is still in PR and unreleased? Just wondering.